garrysmod/html/js/menu/control.Servers.js

```diff
@@ -230,10 +230,7 @@ function AddServer( type, id, ping, name, desc, map, players, maxplayers, botpla
 		workshopid:		workshopid
 	};
 
-	data.hasmap = DoWeHaveMap( data.map );

```

Comment:
`sv` and `data` aren't global variables. This data object is added to the `servers` array in the gamemode object. When being looped over, `ServerClass` is called, with this data object passed in as a parameter.

Best thing is to test it and see if the missing maps are coloured red or not.